pages/terms.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page-no-toc
+title: "Terms of Use"
+permalink: terms
+---
+
+# Terms of Use
+
+## URI Persistence Policy
+
+Last modified: 2021-01-25
+
+When information is made available on the Web, it is important for the integrity of the Web, and the society based upon it, that the URIs used to reference information be used well into the future, and that the information persist as identified.
+
+To this end, the Solid Project hosts make the following pledge: that as far as they are able, for resources on the solidproject.org Web site which are declared (see below) to be persistent.
+
+Pledge:
+
+* The hosts will ensure that persistent resources continue to be available throughout the life of the Solid Project;
```

Comment:
"throughout the life of the Solid Project
" is not enough.And if the project is wrapped up will find someone to take over the domain and serve it indefinitely.

Reply:
Is that pledge already covered in the third point:

"Should the Solid Project be disbanded [..] the original https://solidproject.org web site will be handed over for management to another organization only if that organization pledges to this policy or one considered more persistent." ?